### PR TITLE
Add d-block to bootstrap 4 alerts

### PR DESCRIPTION
**Pull request by normisg**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #26869
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

**Comment by chapterjason** [comment]
Is there any reason why a `span` is used to display the alerts? When I look at the docs of bootstrap, I can clearly see that the `div` tag are used for that.

https://getbootstrap.com/docs/4.0/components/alerts/

Using `d-block` is for me really odd.

**Comment by normisg** [comment]
Yes, it was changed in https://github.com/symfony/symfony/commit/5dcaaadb5e8c01148b67d009c95a5118849f9a55 to "Make sure form errors is valid HTML".
Seems unnecessary to me too.

**Comment by vudaltsov** [comment]
The reason for this is that now error blocks might appear inside `<label>` tags. You cannot have `<div>` inside a label.

File: src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig [hunk 1]
```diff
@@ -283,7 +283,7 @@
 
 {% block form_errors -%}
     {%- if errors|length > 0 -%}
-        <span class="{% if form is not rootform %}invalid-feedback d-block{% else %}alert alert-danger{% endif %}">
+        <span class="{% if form is not rootform %}invalid-feedback d-block{% else %}alert alert-danger d-block{% endif %}">
```
Comment (vudaltsov) [inline src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig]: Let's move the `d-block` class outside the `{% if %}` statement to have it only once in the line

**Review by vudaltsov — CHANGES_REQUESTED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @normisg.

**Pull request merged**